lib/IRGen/GenCall.cpp

```diff
@@ -1317,7 +1317,15 @@ void SignatureExpansion::expandExternalSignatureTypes() {
     paramTys.push_back(clangCtx.VoidPtrTy);
     break;
 
-  case SILFunctionTypeRepresentation::CXXMethod:
+  case SILFunctionTypeRepresentation::CXXMethod: {
```

Comment:
Could you please add a comment to this method  `expandExternalSignatureTypes` to mention that it also expands the parameter types for C++ methods.

Reply:
For sure!